package.json

```diff
@@ -96,7 +96,7 @@
     "vscode:bundle": "npm run vscode:bundle:debugger && npm run vscode:bundle:extension",
     "vscode:bundle:debugger": "lerna run bundle:debugger",
     "vscode:bundle:extension": "lerna run bundle:extension",
-    "vscode:package": "npm run vscode:bundle:debugger && npm run vscode:bundle:extension && lerna run vscode:package --concurrency 1 && npm run reformat",
```

Comment:
running the command vscode:bundle wrapped within capture-esbuild-warnings.js to report any unbundle-able dependencies during bundling.